client/wp-admin-scripts/print-shipping-label-banner/lib/banner-tracks.js

```diff
@@ -0,0 +1,39 @@
+import { recordEvent } from 'lib/tracks';
+import { getSetting } from '@woocommerce/wc-admin-settings';
+import apiFetch from '@wordpress/api-fetch';
+const JETPACK_NAMESPACE = '/jetpack/v4';
+
+async function getPluginsInformation() {
+	let isJetpackConnected = false;
+	const url = JETPACK_NAMESPACE + '/connection';
+	const result = await apiFetch( {
```

Comment:
Optional: If we use the wc-api here, we have a selector `isJetpackConnected()` that can check for a Jetpack connection.

Reply:
Thanks @joshuatf . I will look into how to I can switch to wc-api here.

Reply:
Feel free to ping me if you need any ideas @harriswong

Reply:
Switched to `wc-api`